src/utilities/layer-from-json.ts

```diff
@@ -0,0 +1,22 @@
+import * as layer from '../layer'
+import { get } from 'lodash'
+import { ILayer, ILayerJSON } from '../layer/base-layer';
+
+export default function layerFromJSON(jsonLayer: ILayerJSON, inputLayer1?: ILayer) { 
```

Comment:
we have an extra unused parameter `inputLayer1` here.

Reply:
This is here because when this function is used in feed-forward.ts it requires two arguments when implemented in typescript, it did not work before in JS so I have no idea why it was there in the first place. Now it is here to not to make changes to the feed-forward.ts file.